dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -0,0 +1,49 @@
+class EvaluateRubricJob < ApplicationJob
+  queue_as :default
+
+  self.queue_adapter = :delayed_job
```

Comment:
see https://github.com/code-dot-org/code-dot-org/pull/54149 for why this is now needed